vcs/src/main/java/org/openjdk/skara/vcs/git/GitCommits.java

```diff
@@ -135,10 +135,28 @@ public void close() throws IOException {
             }
         }
 
+        Exception exception = null;
+
         for (var i = 0; i < processes.size(); i++) {
             var p = processes.get(i);
             var command = commands.get(i);
-            close(p, command);
+            try {
+                close(p, command);
+            } catch (IOException | RuntimeException e) {
```

Comment:
What kind of runtime exception are you expecting here?

Reply:
I suggested catching it mostly to be defensive. The Skara code base uses RuntimeExceptions quite a lot. In this case, there is no obvious place where one would be thrown though

Reply:
> I suggested catching it mostly to be defensive. The Skara code base uses RuntimeExceptions quite a lot. In this case, there is no obvious place where one would be thrown though

I've had a look at those; I see what you mean.

A separate nit for later is calling those exceptions _errors_; this is not conventional in Java:

    class GitHubApplicationError extends RuntimeException

"Error" strongly suggests a subclass of `java.lang.Error`. If you need some other word, but "Exception" is undesirable, you may want to call it "Failure".

Reply:
I think we can improve it later, but not in this pr.

Reply:
I see you have already integrated the PR; okay.

Erik, I couldn't see where those Skara-specific runtime exceptions could be thrown from in this particular case, but maybe you see it as some sort of future proofing?

Reply:
> Erik, I couldn't see where those Skara-specific runtime exceptions could be thrown from in this particular case, but maybe you see it as some sort of future proofing?

> In this case, there is no obvious place where one would be thrown though

I meant that there is no explicit throwing of RuntimeException in this code, but if we would ever get an NPE or similar, I think we should still make sure all processes are killed.